config/detekt/detekt.yml

```diff
@@ -0,0 +1,604 @@
+build:
+  maxIssues: 1
+  excludeCorrectable: false
+  weights:
+    # complexity: 2
+    # LongParameterList: 1
+    # style: 1
+    # comments: 1
```

Comment:
nit. I don't think that these commented out configurations are needed.
I would cleanup this config file to only contain what you need.

**NOTE**:  Keeping the comments change the type `weights` from a map to a null value
The same might be applied to `#/processors/exclude`, `#/console-reports/exclude`, etc.

Reply:
I'd rather keep this file as it is as this is the output of `./gradlew plugin:detektGenerateConfig`.

I've just reapplied the customization we had on `plugin/detekt-config.yml` (now deleted).
Modifying this file will make diffing in the future harder.

Reply:
👍